pgbouncer/metadata.csv

```diff
@@ -8,6 +8,8 @@ pgbouncer.stats.total_query_time,rate,,microsecond,,Time spent by pgbouncer acti
 pgbouncer.stats.total_transaction_time,rate,,microsecond,,Time spent by pgbouncer in transactions,0,pgbouncer,transaction time
 pgbouncer.stats.avg_req,gauge,,request,second,The average number of requests per second in last stat period,0,pgbouncer,requests per second
 pgbouncer.stats.avg_query_count,gauge,,query,second,The average number of queries per second in last stat period,0,pgbouncer,queries per second
+pgbouncer.stats.total_wait_time,rate,,microsecond,,"Time spent by clients waiting for a server, in microseconds",0,pgbouncer,query time
```

Comment:
When submitted as a GAUGE, the type in the metadata should be a `gauge`. Long story, but everything is explained here: https://docs.datadoghq.com/developers/metrics/types/?tab=count
 ```suggestion
pgbouncer.stats.total_wait_time,gauge,,microsecond,,"Time spent by clients waiting for a server, in microseconds",0,pgbouncer,total wait time
```